TheGnarCo/gnarails
# Integrate gnareact

**This PR:**

* Integrates Gnareact into Gnarails

**How to test:**

1) Run the Gnarails setup script with `--webpack=react` option

```bash
> rails new app-name -m ./gnarails/gnarly.rb --skip-test-unit --database=postgresql --webpack=react
```

2) `cd app-name`

3) Set up Gnarails instance

```bash
> bundle install
> bin/yarn
> bin/rails db:create
> bin/rails db:migrate
```

4) Ensure JS tests run without error

```bash
> bin/yarn test
```

5) Run the application

_In one terminal tab:_

```bash
> bin/rails s
```

_In another terminal tab_

```bash
> bin/yarn start
```

6) Visit `localhost:3000` in browser

7) You should see "Welcome to Gnarails"
